roles/BCPC-Hadoop-Head-HBase.json

```diff
@@ -4,6 +4,7 @@
   "run_list": [
     "role[Basic]",
     "role[BCPC-Hadoop-Head]",
+    "recipe[bcpc-hadoop::hbase_repl]",
```

Comment:
Do we want to add this to the run list on VM cluster? There are usually no peer clusters on VMs?